fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -7,65 +7,190 @@ package org.mozilla.fenix.tabstray
+        normalTabCountTextModifier = Modifier.padding(bottom = 1.dp)
+    } else {
+        normalTabCountText = NumberFormat.getInstance().format(normalTabCount.toLong())
+        normalTabCountTextModifier = Modifier
+    }
+    val normalTabsContentDescription = if (normalTabCount == 1) {
+        stringResource(id = R.string.mozac_tab_counter_open_tab_tray_single)
+    } else {
+        String.format(
+            stringResource(id = R.string.mozac_tab_counter_open_tab_tray_plural),
+            normalTabCount.toString(),
+        )
+    }
+    val selectedColor = FirefoxTheme.colors.iconActive
+    val inactiveColor = FirefoxTheme.colors.iconPrimaryInactive
 
-        Box(modifier = Modifier.weight(1.0f)) {
-            PrimaryButton(text = "Private tabs") {
-                onTabPageIndicatorClicked(Page.PrivateTabs)
+    Column {
+        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.bottom_sheet_handle_top_margin)))
+
+        Divider(
+            modifier = Modifier
+                .fillMaxWidth(DRAG_INDICATOR_WIDTH_PERCENT)
+                .align(Alignment.CenterHorizontally),
+            color = FirefoxTheme.colors.textSecondary,
+            thickness = dimensionResource(id = R.dimen.bottom_sheet_handle_height),
+        )
+
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(80.dp),
+            horizontalArrangement = Arrangement.spacedBy(16.dp),
+        ) {
+            CompositionLocalProvider(LocalRippleTheme provides DisabledRippleTheme) {
+                TabRow(
+                    selectedTabIndex = selectedPage.ordinal,
+                    modifier = Modifier.width(180.dp),
+                    backgroundColor = Color.Transparent,
+                    contentColor = selectedColor,
+                    divider = {},
+                ) {
+                    Tab(
+                        selected = selectedPage == Page.NormalTabs,
+                        onClick = { onTabPageIndicatorClicked(Page.NormalTabs) },
+                        modifier = Modifier.fillMaxHeight(),
+                        selectedContentColor = selectedColor,
+                        unselectedContentColor = inactiveColor,
+                    ) {
+                        Box {
+                            Icon(
+                                painter = painterResource(
+                                    id = mozilla.components.ui.tabcounter.R.drawable.mozac_ui_tabcounter_box,
+                                ),
+                                contentDescription = normalTabsContentDescription,
+                                modifier = Modifier.align(Alignment.Center),
+                            )
+
+                            Text(
+                                text = normalTabCountText,
+                                modifier = normalTabCountTextModifier.align(Alignment.Center),
+                                color = LocalContentColor.current,
```

Comment:
so `LocalContentColor` is pulling from what is essentially a locally-defined asset (this can be used for colors, themes, dimensions, and several other theming things); think of it like a localized theme override of a specific attribute. In this specific scenario, `LocalContentColor` is being set by `Tab` when that particular `Tab` is being selected/unselected. We have the selected/unselected colors defines above, but utilizing `LocalContentColor` allows for that coloring to happen behind the scenes (it has to be set manually in `Text` because it doesn't passively consume attribute) and `Tab` actually does a fancy color transition between the two states!